src/query/vacuum.c

```diff
@@ -5611,7 +5611,14 @@ vacuum_recover_lost_block_data (THREAD_ENTRY * thread_p)
 VACUUM_LOG_BLOCKID
 vacuum_get_log_blockid (LOG_PAGEID pageid)
 {
-  return ((pageid == NULL_PAGEID) ? VACUUM_NULL_LOG_BLOCKID : (pageid / vacuum_Data.log_block_npages));
+  if (prm_get_bool_value (PRM_ID_DISABLE_VACUUM) || pageid == NULL_PAGEID)
```

Comment:
How about using `vacuum_Is_booted` instead of checking the system parameter? because this function is called very frequently, especially for every DML.

Reply:
or just checking `vacuum_Data.log_block_npages == NULL`

Reply:
@wrlawodms I checked several API function of vacuum and I saw that they all use 'prm_get_bool_value (PRM_ID_DISABLE_VACUUM)' instead of 'vacuum_Is_booted' or 'vacuum_Data.log_block_npages == NULL'. So I choose it as I thought it fit for purpose. However, your suggestion is good for me. So I want to check other reviewer's opinions. What do you(@JoohoK and @eido5) think about it?

Reply:
@hornetmj What you're saying also makes sense. I think we can leave it as you do and see the update performance test from QA system. We can improve it later if needed. Thank you.

Reply:
@wrlawodms OK. I will check the performance test from QA. Thanks to leave your opinion.